scripts/1.4.0/README.md

```diff
@@ -1,25 +1,14 @@
-# Scripts for v1.3.12
-
-### Upgrade (only for Canary)
-
-Required to upgrade the Canary from any version above 1.3.4 to 1.3.12.
-
-This script is needed to be able to discover DBMSs installed with older versions of Desktop correctly. Without running this, you'll see the DBMSs in your projects, but won't be able to start them. The script is only changing Desktop configuration and is not modifying any data.
-
-- Start the upgraded version of Neo4j Desktop.
-- Once it's done loading close the application.
-- Run the script.
-- Start Neo4j Desktop again.
+# Scripts for v1.3.12 (released as v1.4.0)
 
 ### Downgrade
 
-Required to downgrade from 1.3.12 to any older version. The script needs to be executed after closing Neo4j Desktop 1.3.12 and before starting the older version.
+Required to downgrade from 1.3.12 (released as v1.4.0) to any older version. The script needs to be executed after closing Neo4j Desktop 1.3.12 (released as v1.4.0) and before starting the older version.
 
 This script is needed for projects to be discovered correctly by older versions of Desktop. Without it, you won't be able to startup Desktop (as it won't find the project you last opened). The script is removing some symbolic links and metadata used by Desktop, no data is modified by it.
 
 **Projects directory not found**
```

Comment:
does the `Projects directory not found` need to be added to the Canary upgrade script too @nglgzz ? I thought it might not given the script.

Reply:
No it's not needed for the Canary @jk05